Admin/NewsAdmin.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Example\NewsBundle\Admin;
+
+use Sulu\Bundle\AdminBundle\Admin\Admin;
+use Sulu\Bundle\AdminBundle\Navigation\Navigation;
+use Sulu\Bundle\AdminBundle\Navigation\NavigationItem;
+
+class NewsAdmin extends Admin
+{
+    public function __construct($title)
+    {
+        $rootNavigationItem = new NavigationItem($title);
+
+        $section = new NavigationItem('navigation.webspaces');
+
+        $global = new NavigationItem('navigation.global-content');
+        $section->addChild($global);
+
+        $news = new NavigationItem('navigation.news');
+        $news->setAction('example/news');
```

Comment:
The action will be used by the JavaScript application to determine which component should be loaded to display the content for that page.